### PR TITLE
new background, fixes to email label on signup start

### PR DESCRIPTION
**Pull request by jpslav**
Student has a labeled email field now, and other faculty-level non-instructor roles have the same email text as the instructor role.

image

image

image

### CODE REVIEW

**Comment by Fredasaurus** [comment]
Looks good

File: app/assets/javascripts/signup/email-value.coffee [hunk 1]
```diff
@@ -27,6 +27,7 @@ class OX.Signup.EmailValue
 
 
   setType: (newUserType) ->
+    newUserType = if newUserType == "student" then "student" else "instructor"
     @group.find("[data-audience=\"#{@userType}\"]").hide()
```
Comment (nathanstitt) [inline app/assets/javascripts/signup/email-value.coffee]: Could also be expressed as: `newUserType = 'instructor' unless newUserType is 'student'`

File: app/assets/stylesheets/application_layout.scss [hunk 2]
```diff
@@ -9,7 +9,7 @@ body {
   background-color: $application_background_color;
   color:            $os_gray;
   margin-top:       0px;
-  background: image_url('home-bg.jpg') no-repeat center center fixed;
+  background: image_url('bg-login.jpg') no-repeat center center fixed;
   background-size: cover;
```
Comment (nathanstitt) [inline app/assets/stylesheets/application_layout.scss]: Should we remove the old `home-bg.jpg` file?  I think this is the only spot that used it.

Comment (nathanstitt) [inline app/assets/stylesheets/application_layout.scss]: `grep -r home-bg.jpg app/assets/stylesheets/*` suggests that's the only spot it's referenced

Comment (jpslav) [inline app/assets/stylesheets/application_layout.scss]: I removed it

**Pull request merged**